obs-studio.install/tools/chocolateyinstall.ps1

```diff
@@ -1,7 +1,10 @@
 ﻿$ErrorActionPreference = 'Stop';
 $toolsDir   = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
-$fileLocation   = Get-Item $toolsDir\*_x32.exe
-$file64Location = Get-Item $toolsDir\*_x64.exe
+$fileLocation   = Join-Path $toolsDir (Get-childitem -Path $toolsDir -Filter *x86.exe -Name)
+$file64Location = Join-Path $toolsDir (Get-childitem -Path $toolsDir -Filter *x64.exe -Name)

```

Comment:
Why is this changed? I'm not opposed to improvement but I don't know why you did this.

Reply:
Unfortunately, `Get-Item` conflicts with the `Get-PackageParameters` that gets the parameter(s). `Get-childitem` works, but only gives the name of the file, so it needs to be joined with the rest of the path.